tests/online_migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -19,6 +19,19 @@ sub run {
     }
     else {
         select_console 'x11';
+        # check screenlock after zypper patch
+        wait_still_screen;
+        send_key "backspace";
+        if (check_screen('screenlock')) {
+            send_key "esc";
+            if (check_screen 'gnome-screenlock-password') {
+                type_password;
+                send_key "ret";
+            }
+        }
+        mouse_hide(1);
+        assert_screen 'generic-desktop';
```

Comment:
utils::wait_boot and first_boot do not contain and not check screenlock.
this reduced snippet came from tests/console/consoletest_finish.pm